### ISSUE TITLE
Aeotec Range Extender Zi

### ISSUE THREAD

**Issue by stijnmeijer** [dresden-elektronik/deconz-rest-plugin#6785]
## Device

- Product name: Range Extender Zi
- Manufacturer: AL001
- Model identifier: WG001-Z01
- Device type : 
  - Other: Range extender

## Screenshots

![afbeelding](https://user-images.githubusercontent.com/7773636/223146976-c9d4fdc2-aa3a-4376-b3a3-a09f23c83d69.png)

### Basic

![afbeelding](https://user-images.githubusercontent.com/7773636/223148947-55a07d3b-049e-4766-a031-f70ca18dfa30.png)

### Identify

![afbeelding](https://user-images.githubusercontent.com/7773636/223149414-cbd7d393-9e15-4a8b-a063-aad455b5e64b.png)
_Note: after pressing Read and/or Exec, the value remains 0_

### OTAU

![afbeelding](https://user-images.githubusercontent.com/7773636/223151969-33c36783-761b-465a-af94-6ee28e967120.png)
![afbeelding](https://user-images.githubusercontent.com/7773636/223152067-e0e2149c-bdc6-4b20-b2c0-9e77a8051d19.png)

All other clusters do not seem to contain any attributes, or are duplicate to the above.

**Comment by Smanar** [comment]
>I'm not sure if this is correct, as it seems too straightforward.

Lol, yes. And it will be the first DDF for range extender

```
{
  "schema": "devcap1.schema.json",
  "manufacturername": "AL001",
  "modelid": "WG001-Z01",
  "product": "Aeotec Range Extender Zi",
  "sleeper": false,
  "status": "Gold",
  "subdevices": [
    {
      "type": "$TYPE_RANGE_EXTENDER",
      "restapi": "/lights",
      "uuid": [
        "$address.ext",
        "0x01"
      ],
      "items": [
        {
          "name": "attr/id"
        },
        {
          "name": "attr/lastannounced"
        },
        {
          "name": "attr/lastseen"
        },
        {
          "name": "attr/manufacturername"
        },
        {
          "name": "attr/modelid"
        },
        {
          "name": "attr/name"
        },
        {
          "name": "attr/swversion"
        },
        {
          "name": "attr/type"
        },
        {
          "name": "attr/uniqueid"
        },
        {
          "name": "state/reachable"
        }
      ]
    }
  ]
}
```

IDK if your device have special feature that can be used in the DDF ?

**Comment by Idaho947** [comment]
Works for me

**Comment by Smanar** [comment]
You have this device too ?

**Comment by Idaho947** [comment]
Yes I bought it. i am a device tester in real life.

**Comment by Smanar** [comment]
And this device haven't special feature ? (So much endpoints and clusters for nothing)

**Comment by Idaho947** [comment]
I don't know. For me it's just an extender

**Comment by wvuyk** [comment]
why to expose an extender to the api if there is no added value then extend signals.....?

**Comment by Idaho947** [comment]
For exemple to check if it's always on the network.

**Comment by wvuyk** [comment]
You will not see any updates in the API to tell the device is still alive? You need to check on the deCONZ map. That is the place to check is a device is still on the network. 
Adding an extender to the api does not any value, there are no controls and there are no updates. Instead apps using the api need to provision ways to ignore these devices.  It really adds no value to the api?

**Comment by Idaho947** [comment]
Ok but if your using a third party app for your domotic use and this third party use the api ?

**Comment by wvuyk** [comment]
Yes, I am maintaining my own app here

**Comment by wvuyk** [comment]
Sorry, my last answer was unclear. I am a writer of a thrid party app. That is also the reason for my earlier answer. Why would a third party app show a device that has no controls and no updates that are of interest for the user? For my app, I hide these devices for the user as the app can not show any assured confirmation it is alive?

**Comment by Smanar** [comment]
Using state/reachable don't work ?

**Comment by Idaho947** [comment]
Yes it works. But in my memories, my app use the api to import and use it.

**Comment by Schmandre** [comment]
Hello,

I just tried to pair the device but can’t see anything in Web UI. Any information how to get that to work?

Using Phoscon in docker
